ccm_web/server/src/api/api.utils.ts

```diff
@@ -0,0 +1,18 @@
+import { HTTPError } from 'got'
+import { APIErrorData } from './api.interfaces'
+
+import baseLogger from '../logger'
+
+const logger = baseLogger.child({ filePath: __filename })
+
+export function handleAPIError (error: unknown): APIErrorData {
```

Comment:
Why has this been moved to a new module?

Reply:
@pushyamig needed access to it in her handler class.

Reply:
Yeah... I did the refactoring this function could be used by all API endpoints